packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -190,27 +191,46 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
           </Form>
         </Panel>
         {!isLoadingPeerProfile && isValidURL && (
-          <Panel header={t('peerList.panels.information')}>
-            <Panel
-              bordered={true}
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
-              }}
-            />
-            <DescriptionList>
-              <DescriptionListItem label={t('peerList.panels.name')}>
-                {profile?.name}
-              </DescriptionListItem>
-              <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
-              </DescriptionListItem>
-            </DescriptionList>
-          </Panel>
+          <>
+            <Panel header={t('peerList.panels.information')}>
+              <Panel
+                bordered={true}
+                style={{
+                  height: '200px',
+                  marginBottom: '20px',
+                  backgroundSize: 'cover',
+                  backgroundImage: `url(${
+                    profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
+                  }`
+                }}
+              />
+              <DescriptionList>
+                <DescriptionListItem label={t('peerList.panels.name')}>
+                  {profile?.name}
+                </DescriptionListItem>
+                <DescriptionListItem label={t('peerList.panels.themeColor')}>
+                  {profile?.themeColor}
+                </DescriptionListItem>
+              </DescriptionList>
+            </Panel>
+            <Panel header={t('peerList.panels.callToAction')}>
+              <DescriptionList>
```

Comment:
I don't think it's necessary to have two DescriptionLists. you can add the two new information to the one above.